aten/src/ATen/native/native_functions.yaml

```diff
@@ -1229,6 +1229,12 @@
 
 - func: conv_transpose3d.input(Tensor input, Tensor weight, Tensor? bias=None, int[3] stride=1, int[3] padding=0, int[3] output_padding=0, int groups=1, int[3] dilation=1) -> Tensor
 
+# out-of-place variant of copy_()
+# This doesn't need to be exposed to python, but is needed for functionalization.
+- func: copy(Tensor self, Tensor src, bool non_blocking=False) -> Tensor
```

Comment:
not immediately, since I didn't want to preemptively add a bunch of new out-of-place ops. But I bet that as functionalization is used more broadly then we'll start to have to add more out-of-place ops. Right now, you'll get a TORCH_WARN if you try to functionalize an inplace op with no out-of-place variant.

Reply:
Do I understand correctly from our offline conversation that this entry is going to go away?

Reply:
That was my understanding: that the plan is to insert ".to().expand_as()" instead of having a copy operator

Reply:
yup- I'll close this PR and add a new one that special cases the `.to().expand_as()` mapping in the odegen.